tests/run.bats

```diff
@@ -545,3 +545,54 @@ function configure_and_check_user() {
 	run_buildah 125 run --user noexist $cid whoami
 	expect_output --substring "unknown user error"
 }
+
+@test "run --runtime --runtime-flag" {
+	skip_if_in_container
+	skip_if_no_runtime
+
+	_prefetch alpine
+
+	# Use seccomp to make crun output a warning message because crun writes few logs.
+	cat > ${TESTDIR}/seccomp.json << _EOF
+{
+    "defaultAction": "SCMP_ACT_ALLOW",
+    "syscalls": [
+        {
+	        "name": "unknown",
+			"action": "SCMP_ACT_KILL"
+	    }
+    ]
+}
+_EOF
+	run_buildah from --security-opt seccomp=${TESTDIR}/seccomp.json --quiet --pull=false --signature-policy ${TESTSDIR}/policy.json alpine
+	cid=$output
+
```

Comment:
I would encourage you to do something like:
```bash
    local found_runtime=
    if [ -n ...]; then
        found_runtime=y
    ...
    if [ -n ...crun]
        found_runtime=y
    ...

    if [[ -n "$found_runtime" ]]; then
        die "Did not find 'runc' nor 'crun' in \$PATH - could not run this test!"
    fi
```
(I know there's already `skip_if_no_runtime`, but I can be very paranoid; and I've seen countless cases where tests pass because they aren't actually running anything).